cmd/podman/kube/play.go

```diff
@@ -265,7 +271,34 @@ func play(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	return kubeplay(reader)
+	if err = kubeplay(reader); err != nil {
+		return err
+	}
+
+	if playOptions.Wait {
+		fmt.Println("Use ctrl+c to clean up")
+		// Stop the the shutdown signal handler so we can actually clean up after a SIGTERM or interrupt
+		if err := shutdown.Stop(); err != nil {
+			return err
+		}
+
+		// Read the kube yaml file again so that a reader can be passed down to the teardown function
+		teardownReader, err := readerFromArg(args[0])
+		if err != nil {
+			return err
+		}
+
+		// Create a channel to catch an interrupt or SIGTERM signal
+		ch := make(chan os.Signal, 1)
+		signal.Notify(ch, os.Interrupt, syscall.SIGTERM)
+		<-ch
+		fmt.Println()
+		fmt.Println("Cleaning up containers, pods, and volumes...")
```

Comment:
Maybe drop the `Println` on the previous line and make this "\nCleaning up containers, pods, and volumes..."

Reply:
fixed